metricbeat/module/jolokia/dynamic/data.go

```diff
@@ -0,0 +1,103 @@
+package dynamic
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+type Entry struct {
+	Request struct {
+		Mbean string `json:"mbean"`
+	}
+	Value map[string]interface{}
+}
+
+// Map responseBody to common.MapStr
+func eventMapping(responseBody []byte, mapping map[string]string, application string, instance string) (common.MapStr, error) {
+
+	debugf("Got reponse body: ", string(responseBody[:]))
+
+	var entries []Entry
+	err := json.Unmarshal(responseBody, &entries)
+
+	if err != nil {
+		return nil, fmt.Errorf("Cannot unmarshal json response: %s", err)
+	}
+
+	debugf("Unmarshalled json: ", entries)
+
+	event := map[string]interface{}{}
+
+	if application != "" {
+		event["application"] = application
```

Comment:
I see, this is really a predefined field. Except for putting it under a fixed namespace I don't have a good idea yet where we could put these and how to document it.

In docker we have a similar problem where several metricsets also have the container info. We use some tricks there to add this info the the evend and have it then under the container metricset. Perhaps we should also introduce such a namespace here. `jolokia.?.application`. Like this we can also document it as it will not be under the dynamic namespace. Some suggestions with what we should replace ?